distributed-exceptions/src/main/java/io/axoniq/distributedexceptions/rest/IssueNewGiftCardDto.java

```diff
@@ -1,12 +1,16 @@
 package io.axoniq.distributedexceptions.rest;
 
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
 /**
  * @author Stefan Andjelkovic
  */
 public class IssueNewGiftCardDto {
     private final int amount;
 
-    public IssueNewGiftCardDto(int amount) {
+    @JsonCreator
+    public IssueNewGiftCardDto(@JsonProperty("amount")  int amount) {
```

Comment:
I believe that if there's one constructor, you can come by with just using the `JsonProperty` annotation and nothing else.